app/src/main/res/layout/onboarding_fragment.xml

```diff
@@ -43,62 +43,61 @@
-          android:layout_height="wrap_content"
-          android:background="@drawable/primary_rounded_button"
-          android:fontFamily="sans-serif-medium"
-          android:minHeight="48dp"
-          android:onClick="@{(v) -> presenter.clickOnGetStarted()}"
-          android:text="@string/get_started"
-          android:textColor="@color/white"
-          android:textSize="14sp"
-          app:layout_constraintBottom_toBottomOf="parent"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
-      </androidx.constraintlayout.widget.ConstraintLayout>
-
-      <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/bottom_container"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:paddingStart="28dp"
-        android:paddingEnd="28dp"
-        android:paddingBottom="16dp"
-        android:visibility="@{viewModel.slideNumber != (viewModel.totalNumberOfSlides - 1) ? View.VISIBLE: View.GONE, default=visible}">
+          android:layout_height="1dp"
+          android:layout_marginBottom="16dp"
+          android:background="@color/oppiaPrimaryText"
+          app:layout_constraintBottom_toTopOf="@+id/slide_dots_container" />
 
         <LinearLayout
           android:id="@+id/slide_dots_container"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
+          android:layout_marginTop="32dp"
+          android:layout_marginBottom="16dp"
+          android:minHeight="48dp"
+          android:gravity="center"
           android:orientation="horizontal"
           app:layout_constraintBottom_toBottomOf="parent"
           app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
+          app:layout_constraintStart_toStartOf="parent" />
+
+        <ImageView
+          android:id="@+id/onboarding_fragment_next_image_view"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginEnd="32dp"
+          android:gravity="center"
+          android:minWidth="48dp"
+          android:minHeight="48dp"
+          android:onClick="@{(v) -> presenter.clickOnNext()}"
+          android:visibility="@{viewModel.slideNumber != 3 ? View.VISIBLE: View.GONE, default=visible}"
+          android:paddingStart="12dp"
+          android:paddingEnd="12dp"
+          android:src="@drawable/ic_arrow_forward_black_24dp"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintTop_toTopOf="@+id/slide_dots_container" />
 
         <TextView
           android:id="@+id/skip_text_view"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
+          android:layout_marginStart="32dp"
```

Comment:
This should be `24dp`.
The skip button width in mock is `32dp` only and in this code it is `48dp` and that means that extra `16dp` is gone into left and right padding for the text by `8dp` each. Therefore, `32-8 = 24dp`

Reply:
This is changed to 20dp for consistency with margin end

Reply:
Ack.